### DISCUSSION TITLE
xml是否可以放在配置中心管理

### DISCUSSION THREAD

**Discussion by znrun** [apolloconfig/apollo#4724] [Q&A]
java对接apollo时，yml和xml放在配置中心，yml文件可以读取成功，xml配置文件修改，应用读取不到？
是配置中心不支持xml读取吗？

**Comment by nisiyong** [comment]
You need to read this [java-sdk-user-guide](https://www.apolloconfig.com/#/zh/usage/java-sdk-user-guide?id=%e4%b8%89%e3%80%81%e5%ae%a2%e6%88%b7%e7%ab%af%e7%94%a8%e6%b3%95), the java client supports xml format.
